subsys/nrf_security/src/drivers/cracen/cracenpsa/src/cracen_mac_hmac.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <psa/crypto.h>
+#include "common.h"
+#include "cracen_psa_primitives.h"
+
+/**
+ * @brief Set up an HMAC operation.
+ *
+ * This function initializes an HMAC operation with the provided key and algorithm.
+ *
+ * @param[in,out] operation       Pointer to the HMAC operation context to be initialized.
+ * @param[in]     attributes      Pointer to the key attributes structure.
+ * @param[in]     key_buffer      Pointer to the buffer containing the key material.
+ * @param[in]     key_buffer_size Size of the key buffer in bytes.
+ * @param[in]     alg             The HMAC algorithm to be used.
+ *
+ * @return PSA_SUCCESS on success, or an appropriate error code on failure.
+ */
+
```

Comment:
nit: We don't use newlines between the comments/descriptions and the function as far as I can tell